core/controllers/editor.py

```diff
@@ -501,20 +717,33 @@ def get(self, exploration_id):
 class ExplorationRevertHandler(EditorHandler):
     """Reverts an exploration to an older version."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'current_version': {
+                'schema': {
+                    'type': 'int'
+                }
+            },
+            'revert_to_version': {
+                'schema': {
+                    'type': 'int'
+                }
+            }
```

Comment:
Any other rules for version numbers? Maybe >0? And maybe current>revert_to?

Reply:
Added a validator to check version greater than 0, but are we expecting to check current_version > revert_version in param validation layer ?
If yes, current architecture is designed to add validators for a single argument but in this scenario we have handle two different arguments simultaneously.
Lokking for your suggestions.

Reply:
Yeah, the second case is probably for the function itself and not for the validator, can you check that the second check is somewhere in the code?